gdx/src/com/badlogic/gdx/scenes/scene2d/ui/ParticleEffectActor.java

```diff
@@ -0,0 +1,100 @@
+package com.badlogic.gdx.scenes.scene2d.ui;
+
+import com.badlogic.gdx.files.FileHandle;
+import com.badlogic.gdx.graphics.g2d.Batch;
+import com.badlogic.gdx.graphics.g2d.ParticleEffect;
+import com.badlogic.gdx.graphics.g2d.TextureAtlas;
+import com.badlogic.gdx.scenes.scene2d.Actor;
+
+/**
+ * ParticleEffectActor holds an {@link ParticleEffect} to use in Scene2d applications.
+ * The particle effect is positioned in the centered in the ParticleEffectActor. Its bounding box
+ * is not limited to the size of this actor.
+ */
+public class ParticleEffectActor extends Actor {
+    private final ParticleEffect particleEffect;
+    float lastDelta;
+    private boolean isComplete = true;
```

Comment:
Shouldn't it be `false`?

Reply:
isComplete is false while the effect is running in this implementation, and it does not autostart

Reply:
yep, I think there are 3 different states, "Not started", "Running" and "Complete", we want the property to tells us if it's "Running".